climada_petals/hazard/tc_tracks_forecast.py

```diff
@@ -564,7 +564,7 @@ def _cxml_to_df(
         import io
 
         if xsl_path is None:
-            xsl_path = CXML2CSV_XSL
+            xsl_path = str(CXML2CSV_XSL)
```

Comment:
it was crashing before on my machine.

Reply:
and @emanuel-schmid here is the other. Do we want to constrain the lxml package to > 4.8.0 releases or not. Depending on the choice we would need that string transformation or not.

Reply:
As lxml is a dependency of only pandas-datareader so far, I don't mind much pinning it to 4.8. but if you do, please have a second look at all the arguments that are of type str but intrinsically pathes and those that can be either a str or a Path.
Imho, allowing a str where a Path is expected should be avoided and if possible restricted to methods that inherit their signature. To be on the future looking side, make them just Path'es wherever possible. 🧐

Reply:
Since @emanuel-schmid is pushing for a top-level import, and since the lxml version currently resolved by conda is something before 4.8.0, I suggest we leave in the string transformation for now, maybe with a `# TODO drop string transformation once lxml >= 4.8.0`.